apps/src/templates/WidgetContinueButton.jsx

```diff
@@ -0,0 +1,32 @@
+/** @file Button that continues to the next puzzle when clicked */
+/* global dashboard */
+import React from 'react';
+import i18n from '@cdo/locale';
+
+const ContinueButton = React.createClass({
+  getInitialState() {
+    return {submitting: false};
+  },
+
+  onClick() {
+    this.setState({submitting: true});
+    dashboard.dialog.processResults(willRedirect => {
+      if (!willRedirect) {
+        this.setState({submitting: false});
+      }
+    });
+  },
+
+  render() {
+    return (
+      <button
+        className="btn btn-primary pull-right"
+        disabled={!!this.state.submitting}
+        onClick={this.state.submitting ? null : this.onClick}
```

Comment:
I think `pull-right` is something that will stop working when we move to the next version of bootstrap. This is probably ok to add, but just noting that it would be nice as a cabal to have a plan on what we're going to do w.r.t bootstrap in apps.